docs/source/tutorials/regression.ipynb

```diff
@@ -0,0 +1,521 @@
+      "text/plain": [
+       "<Figure size 1400x500 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Generate true errors\n",
+    "true_errors_index = np.where(data[\"grade\"] != data[\"true_grade\"])[0]\n",
+    "plot_data(\n",
+    "    data_x=data[\"exam_3\"],\n",
+    "    data_y=data[\"grade\"],\n",
+    "    circles=true_errors_index,\n",
+    "    title=\"Messy Regression dataset\",\n",
+    "    xlabel=\"exam_3 feature\",\n",
+    "    ylabel=\"grade (Y value)\",\n",
+    ")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the above plot, `grade (Y value)` is plotted against one of the features in the dataset (`exam_3`). We have circled the examples that were considered as `true_error` in **Red** . "
+   ]
+  },
```

Comment:
print the first few rows of subset of dataframe restricted to where true_grade != grade, in order to show some of the label errors.

After that extract the `true_grade` column out of `data` here, not in the XGBoost code below.